### PR TITLE
BaSP-MR-168: [Admin] Create new admin profile screen

### PR DESCRIPTION
**Pull request by lorenzomauro59**
- Build a form to display the admin's information
- Enable editing of the admin's information.
- Adjust and correct the styles.

### CODE REVIEW

**Comment by lorenzomauro59** [comment]
[[Admin : Profile] Create new admin profile screen](https://trello.com/c/cchs6WAZ/168-admin-profile-create-new-admin-profile-screen)

File: src/Components/Admin/Profile/index.jsx [hunk 1]
```diff
@@ -84,27 +71,21 @@ const AdminProfile = () => {
...
+
+  const handleDisableEditMode = () => {
+    setEditMode(false);
     reset();
   };
 
   return (
     <div className={styles.formContainer}>
-      <h2 className={styles.formTitle}>{id ? 'Update Admin' : 'Add Admin'}</h2>
+      <h2 className={styles.formTitleTwo}>user data</h2>
```
Comment (fernandofernandezuy) [inline src/Components/Admin/Profile/index.jsx]: I think here should said admin data

**Review by fernandofernandezuy — APPROVED** [review]
Great Job Loren!

**Review by AleMac17 — APPROVED** [review]

**Review by ale-schmidt — APPROVED** [review]
All looks and works great, congrats!

**Pull request merged**